shared/AppInsightsCore/src/JavaScriptSDK/HelperFuncs.ts

```diff
@@ -658,13 +660,12 @@ export function objDefineAccessors<T>(target: any, prop: string, getProp?: () =>
     return false;
 }
 
-export function objFreeze<T>(value: T): T {
-    return _objFreeze ? _objFreeze(value) as T : value;
+function _doNothing<T>(value: T): T {
+    return value;
 }
 
-export function objSeal<T>(value: T): T {
-    return _objSeal ? _objSeal(value) as T : value;
-}
+export const objFreeze: <T>(value: T) => T = _objFreeze || _doNothing;
```

Comment:
More perf improvements to avoid an extra function call objFreeze() => Object.freeze().

As with isArray, these are now an aliases to the available native function or a dummy one that does nothing. This also is a minor minification improvement as well.